gb-dbg/src/debugger/breakpoints.rs

```diff
@@ -181,4 +181,5 @@ fn is_valid_expression(expr: &str) -> bool {
     expr.chars().all(|c| {
         c.is_alphanumeric() || c.is_whitespace() || c == '=' || c == '>' || c == '<' || c == '!'
     }) && !expr.is_empty()
+        && expr.split_whitespace().count() == 3
```

Comment:
https://github.com/Geal/nom/blob/main/doc/nom_recipes.md#wrapper-combinators-that-eat-whitespace-before-and-after-a-parser